### PR TITLE
Release/2.29.0

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,37 @@
...
+  ([6ea8dd2](https://github.com/tinkoff/taiga-ui/commit/6ea8dd2e122a9083aec133d1c0c37c90ddb4d0f9))
+
+### Bug Fixes
+
+- **addon-doc:** fix vertical scrollbar height for root content
+  ([#1148](https://github.com/tinkoff/taiga-ui/issues/1148))
+  ([128295a](https://github.com/tinkoff/taiga-ui/commit/128295a340c60f4181cd9f6abd3666cd8516cf49))
+- **kit:** `InputPhoneInternation` fix countries ([#1147](https://github.com/tinkoff/taiga-ui/issues/1147))
+  ([eb7829f](https://github.com/tinkoff/taiga-ui/commit/eb7829feb5c39b5b0236f59593567cf0b4fa145f))
+- **kit:** `InputInline` fix hiding first charcter in safari ([#1138](https://github.com/tinkoff/taiga-ui/issues/1138))
```
Comment (MarsiBarsi) [inline CHANGELOG.md:32]: typo: character

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,37 @@
...
+- **addon-doc:** `PieChart`/`RingChart` add animations ([#1133](https://github.com/tinkoff/taiga-ui/issues/1133))
+  ([1803970](https://github.com/tinkoff/taiga-ui/commit/1803970cd0cc6b8e3da86816d64cf82afbe0ed35))
+- **cdk:** use ReplaySubject for TuiDestroyService
+  ([1c00e78](https://github.com/tinkoff/taiga-ui/commit/1c00e78468d3518d83bfa464846ddd3226c1e4f9))
+- **kit:** `InputDate` add full support of `TUI_DATE_FORMAT` ([#1146](https://github.com/tinkoff/taiga-ui/issues/1146))
+  ([52d6472](https://github.com/tinkoff/taiga-ui/commit/52d647257b71409a407e8c00cec85cfc93c64970))
+- **kit:** `InputDateRange` add full support of `TUI_DATE_FORMAT`
+  ([#1155](https://github.com/tinkoff/taiga-ui/issues/1155))
+  ([f6c8d08](https://github.com/tinkoff/taiga-ui/commit/f6c8d080139bba3d55d8608311b9aca594aca4de))
+- **kit:** `InputDateTime` add full support of `TUI_DATE_FORMAT`
```
Comment (MarsiBarsi) [inline CHANGELOG.md:21]: why does it duplicate two points before? 🤔

File: CHANGELOG.md (hunk 1 above)
Comment (vladimirpotekhin) [reply CHANGELOG.md]: thanks 👍

File: CHANGELOG.md (hunk 2 above)
Comment (vladimirpotekhin) [reply CHANGELOG.md:21]: no duplicates, these are different components

**Review by MarsiBarsi — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Pull request merged**